src/garage/torch/embeddings/recurrent_encoder.py

```diff
@@ -0,0 +1,111 @@
+# pylint: disable=attribute-defined-outside-init
+"""A recurrent network with LSTM for encoding context of RL tasks."""
+
+import torch
+from torch import nn
+
+from garage.torch.modules import MLPModule
+
+
+class RecurrentEncoder(MLPModule):
```

Comment:
It feels pretty weird that this inherits from `MLPModule`, since it's not obvious that the MLP layers will be used to encode the input, then the output layer will be used to encode the final output layer. I think this would probably be cleaner by just making MLPModule a field?

It's not really a must-fix issue though.

Reply:
yeah this is pretty strange. what do you use from MLPModule that you wouldn't get from module?

perhaps you actually just want your recurrent encoder to have-a MLPModule (rather than be-a MLPModule)